docs/zh_cn/datasets/det.md

```diff
@@ -53,13 +42,129 @@
 
 ## 准备步骤
 
+## CTW1500
+
+- 第一步: 运行以下命令, 从 [github](https://github.com/Yuliang-Liu/Curve-Text-Detector)下载`train_images.zip`, `test_images.zip`, `train_labels.zip`, `test_labels.zip`等四个文件。
+
+  ```bash
+  mkdir ctw1500 && cd ctw1500
+  mkdir imgs && mkdir annotations
+
+  # 下载并配置标注
+  cd annotations
+  wget -O train_labels.zip https://universityofadelaide.box.com/shared/static/jikuazluzyj4lq6umzei7m2ppmt3afyw.zip
+  wget -O test_labels.zip https://cloudstor.aarnet.edu.au/plus/s/uoeFl0pCN9BOCN5/download
+  unzip train_labels.zip && mv ctw1500_train_labels training
+  unzip test_labels.zip -d test
+  cd ..
+  # 下载并配置数据
+  cd imgs
+  wget -O train_images.zip https://universityofadelaide.box.com/shared/static/py5uwlfyyytbb2pxzq9czvu6fuqbjdh8.zip
+  wget -O test_images.zip https://universityofadelaide.box.com/shared/static/t4w48ofnqkdw7jyc4t11nsukoeqk9c3d.zip
+  unzip train_images.zip && mv train_images training
+  unzip test_images.zip && mv test_images test
+  ```
+
+- 第二步: 运行以下命令，生成 `instances_training.json` 和 `instances_test.json`:
+
+  ```bash
+  python tools/data/textdet/ctw1500_converter.py /path/to/ctw1500 -o /path/to/ctw1500 --split-list training test
+  ```
+
+- 所产生的目录配置应如下:
+
+  ```text
+  ├── ctw1500
+  │   ├── imgs
+  │   ├── annotations
+  │   ├── instances_training.json
+  │   └── instances_val.json
+  ```
+## ICDAR 2011 (原生数位图像 Born-Digital Images)
+
+- 第一步: 运行以下命令, 从[下载地址](https://rrc.cvc.uab.es/?ch=1&com=downloads) `Task 1.1: Text Localization (2013 edition)`下载`Challenge1_Training_Task12_Images.zip`, `Challenge1_Training_Task1_GT.zip`, `Challenge1_Test_Task12_Images.zip`, 和 `Challenge1_Test_Task1_GT.zip`。
+
+  ```bash
+  mkdir icdar2011 && cd icdar2011
+  mkdir imgs && mkdir annotations
+
+  # 下载 ICDAR 2011
+  wget https://rrc.cvc.uab.es/downloads/Challenge1_Training_Task12_Images.zip --no-check-certificate
+  wget https://rrc.cvc.uab.es/downloads/Challenge1_Training_Task1_GT.zip --no-check-certificate
+  wget https://rrc.cvc.uab.es/downloads/Challenge1_Test_Task12_Images.zip --no-check-certificate
+  wget https://rrc.cvc.uab.es/downloads/Challenge1_Test_Task1_GT.zip --no-check-certificate
+
+  # 下载并配置数据
+  unzip -q Challenge1_Training_Task12_Images.zip -d imgs/training
+  unzip -q Challenge1_Test_Task12_Images.zip -d imgs/test
+  # 下载并配置标注
+  unzip -q Challenge1_Training_Task1_GT.zip -d annotations/training
+  unzip -q Challenge1_Test_Task1_GT.zip -d annotations/test
+
+  rm Challenge1_Training_Task12_Images.zip && rm Challenge1_Test_Task12_Images.zip && rm Challenge1_Training_Task1_GT.zip && rm Challenge1_Test_Task1_GT.zip
+  ```
+
+- 第二步: 运行以下命令，生成 `instances_training.json` 和 `instances_test.json`:
+
+  ```bash
+  python tools/data/textdet/ic11_converter.py PATH/TO/icdar2011 --nproc 4
+  ```
+
+- 运行以上命令后, 所产生的目录配置应如下:
+
+  ```text
+  │── icdar2011
+  │   ├── imgs
+  │   ├── instances_test.json
+  │   └── instances_training.json
+  ```
+
+## ICDAR 2013 (聚焦场景文本 Focused Scene Text)
+
```

Comment:
We may do not need to translate the specific dataset name here, just keep `Focused Scene Text`